src/inflators/loop-animation.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  addComponent,
+  hasComponent
+} from "bitecs";
+import {
+  AnimationAction,
+  AnimationClip,
+  AnimationMixer,
+  LoopRepeat
+} from "three";
+import {
+  LoopAnimation,
+  LoopAnimationData,
+  MixerAnimatable,
+  MixerAnimatableData,
+  Object3DTag
+} from "../bit-components";
+import { HubsWorld } from "../app";
+
+// TODO: Type definition should be done in bit-components?
+type MixerAnimatableData = Map<number, {
+  mixer: AnimationMixer,
+  animations: Array<AnimationClip>
+}>;
+type LoopAnimationSata = Map<number, Array<AnimationAction>>;
+
+export type LoopAnimationParams = {
+  activeClipIndex: number;
+  paused: boolean;
+  startOffset: number;
+  timeScale: number;
+  // TODO: Do we need to keep supporting the following two params?
+  // DEPRECATED: Use activeClipIndex instead since animation
+  //             names are not unique
+  clip?: string;
```

Comment:
It looks like Blender currently uses `clip` a comma separated list of names. Spoke uses `activeClipIndices`. I don't see anything using `activeClipIndex` but I bet there are old Spoke scenes that use it. We probably need to support all 3 for now :(

Lets discuss which one we want to be the canonical representation (probably `activeClipIndicies`) and then handle converting the others to that as early as possible in the flow, so the rest of the code does not need to think about it.

We have discussed in other places moving this sort of legacy migration all to an earlier point in GLTF loading after we completely remove gltf-model-plus, so we should add a TODO about that part of the code here.